guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -13,66 +15,31 @@ ifdef::katello,satellite[]
 The runtime size was measured with {RHEL} 6, 7, and 8 repositories synchronized.
 endif::[]
 
-ifdef::foreman-el,katello,satellite[]
 == [[storage-el-8]]{EL} 8
 
-.Storage Requirements for a {ProjectServer} Installation
+.Storage Requirements for {SmartProxyServer} Installation
```

Comment:
This is a common module and results in wrong instructions. It is included in the Installing Server guide.